src/components/SourceImage.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Stack, Typography } from '@mui/material';
+import { ImagePicker } from './ImagePicker';
+import { IS_MOBILE } from '~/domain/utils/isMobile';
+import type { ImageEffectResult } from '~/domain/types';
+import { sliderParam } from '~/params';
+import { DEFAULT_FPS, MAX_FRAMES } from '~/config';
+import { logger } from '~/domain/utils';
+
+const fpsParam = sliderParam({
+  name: 'Final Gif Frames per Second',
+  defaultValue: DEFAULT_FPS,
+  min: 1,
+  max: 60,
+});
+
+const frameCountParam = sliderParam({
+  name: 'Final Gif Frame Count',
+  defaultValue: 0,
+  min: 1,
+  max: MAX_FRAMES,
+});
+
+interface SourceImageProps {
+  baseImage?: ImageEffectResult;
+  fps: number;
+  onImageChange: (
+    baseImage: ImageEffectResult,
+    fname: string,
+    fps: number,
+  ) => void;
+  onFpsChange: (fps: number) => void;
+  onFrameCountChange: (frameCount: number) => void;
+  setAlert: (alert: { severity: 'error' | 'warning'; message: string }) => void;
+}
+
+export const SourceImage: React.FC<SourceImageProps> = ({
+  baseImage,
+  fps,
+  onImageChange,
```

Comment:
Might be worth having a single change event that passes what changed?  image, fps, framecount?